src/main/java/org/prebid/server/analytics/reporter/liveintent/LiveIntentAnalyticsReporter.java

```diff
@@ -0,0 +1,195 @@
+                    .setParameter("bidId=", notificationEvent.getBidId())
+                    .build()
+                    .toString();
+            return httpClient.get(url, properties.getTimeoutMs()).mapEmpty();
+        } catch (URISyntaxException e) {
+            logger.error("Error composing url for notification event: {}", e.getMessage());
+            return Future.failedFuture(e);
+        }
+    }
+
+    private Optional<PbsjBid> buildPbsjBid(
+            BidRequest bidRequest,
+            BidResponse bidResponse,
+            Bid bid,
+            boolean isEnriched,
+            Float treatmentRate,
+            Long timestamp) {
+        return bidRequest.getImp().stream()
+                .filter(impItem -> impItem.getId().equals(bid.getImpid()))
+                .map(imp ->
+                        PbsjBid.builder()
+                                .bidId(bid.getId())
+                                .price(bid.getPrice())
+                                .adUnitId(imp.getTagid())
+                                .enriched(isEnriched)
+                                .currency(bidResponse.getCur())
+                                .treatmentRate(treatmentRate)
+                                .timestamp(timestamp)
+                                .partnerId(properties.getPartnerId())
+                                .build()
+                )
+                .findFirst();
+    }
+
+    private Future<Void> processAuctionEvent(AuctionContext auctionContext) {
+        if(auctionContext.getBidRequest() == null) {
+            return Future.failedFuture(new PreBidException("Bid request should not be empty"));
+        }
+
+        if(auctionContext.getBidResponse() == null) {
+            return Future.failedFuture(new PreBidException("Bid response should not be empty"));
+        }
+
+        final BidRequest bidRequest = auctionContext.getBidRequest();
+        final BidResponse bidResponse = auctionContext.getBidResponse();
+        final Optional<ExtRequestPrebid> requestPrebid = Optional.ofNullable(bidRequest.getExt())
+                .flatMap(ext -> Optional.of(ext.getPrebid()));
+
+        final Optional<Activity> activity = getActivities(auctionContext);
+        final boolean isEnriched = isEnriched(activity);
+        final Float treatmentRate = getTreatmentRate(activity);
+        final Long timestamp = requestPrebid.map(ExtRequestPrebid::getAuctiontimestamp).orElse(0L);
+
+        final List<PbsjBid> pbsjBids = CollectionUtils.emptyIfNull(bidResponse.getSeatbid()).stream()
+                .map(SeatBid::getBid)
+                .flatMap(Collection::stream)
+                .map(bid -> buildPbsjBid(bidRequest, bidResponse, bid, isEnriched, treatmentRate, timestamp))
+                .filter(Optional::isPresent)
+                .map(Optional::get)
+                .toList();
+
+        try {
+            return httpClient.post(
+                    new URIBuilder(URI.create(properties.getAnalyticsEndpoint()))
```

Comment:
No need for `URI.create`

Reply:
Done in https://github.com/prebid/prebid-server-java/pull/4154/commits/58fc8c0e317de0aa062fddda86b40ee1758acca6